internal/handlers/handlers.go

```diff
@@ -1 +1,67 @@
 package handlers
+
+import (
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+)
+
+
+func HtmlHandler(w http.ResponseWriter, r *http.Request) {
+	http.ServeFile(w,r, "index.html")
+}
+
+
+func UploadHandler(w http.ResponseWriter, r *http.Request) {
+	
+	err := r.ParseMultipartForm(10 << 20) 
+	if err != nil {
+		http.Error(w, "Ошибка парсинга", http.StatusInternalServerError)
+		return
+	}
+
+	file, handler, err := r.FormFile("myFile") 
+	if err != nil{
+		http.Error(w, "ошибка при получении файла", http.StatusInternalServerError)
+		return
+	}
+	defer file.Close()
+
+	data, err := io.ReadAll(file) 
+	if err != nil{
+		http.Error(w, "ошибка чтения файла", http.StatusInternalServerError)
```

Comment:
Мы тут тоже хорошие, а клиент плохой
```suggestion
		http.Error(w, "ошибка чтения файла", http.StatusBadRequest)
```